mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -161,14 +206,22 @@ def __init__(self,
                  reduction='mean',
                  class_weight=None,
                  loss_weight=1.0,
-                 loss_name='loss_ce'):
+                 loss_name='loss_ce',
+                 avg_non_ignore=False):
         super(CrossEntropyLoss, self).__init__()
         assert (use_sigmoid is False) or (use_mask is False)
         self.use_sigmoid = use_sigmoid
         self.use_mask = use_mask
         self.reduction = reduction
         self.loss_weight = loss_weight
         self.class_weight = get_class_weight(class_weight)
+        self.avg_non_ignore = avg_non_ignore
+        if not self.avg_non_ignore:
```

Comment:
This warning is a little bit confusing. 
We should warn users when `avg_non_ignore = True`, the loss will be slight larger blablabla...

Reply:
Some users prefer `avg_non_ignore` defaults to `True`, as they think there might be a bug in the original loss calculation. This warning works for reminding them to open `avg_non_ignore`.

I think msg should be refined to ba more straightforwardly.